org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/text/correction/RefactorProposalUtility.java

```diff
@@ -315,20 +315,34 @@ private static boolean supportsExtractVariable(IInvocationContext context) {
 		return true;
 	}
 
-	public static CUCorrectionProposal getExtractVariableAllOccurrenceProposal(CodeActionParams params, IInvocationContext context, boolean problemsAtLocation, Map formatterOptions, boolean returnAsCommand) throws CoreException {
+	public static CUCorrectionProposal getExtractVariableAllOccurrenceProposal(CodeActionParams params, IInvocationContext context, boolean problemsAtLocation, Map formatterOptions, boolean returnAsCommand, boolean inferSelectionSupport) throws CoreException {
```

Comment:
Since this method is a public static method, it's kind of API, i prefer not to change its signature frequently, but add a new overriding method with your new extra parameter there.

Reply:
Make sense. Fixed.